ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateStreamCommand.java

```diff
@@ -35,15 +35,14 @@
 
   @Override
   public DdlCommandResult run(final MutableMetaStore metaStore) {
-    if (registerTopicCommand != null) {
-      try {
-        registerTopicCommand.run(metaStore);
-      } catch (KsqlException e) {
-        final String errorMessage =
-                String.format("Cannot create stream '%s': %s", topicName, e.getMessage());
-        throw new KsqlException(errorMessage, e);
-      }
+    try {
+      createTopic(metaStore);
+    } catch (KsqlException e) {
+      final String errorMessage =
```

Comment:
I think we can remove this try/catch block.  `createTopic` already creates a very similar error message.

`createTopic` will throw:
> [stream|table] with name 'foo' already exists
Where it picks `stream` or `table` depending on what is already registered

`run` will throw:
> Cannot create stream 'foo': [stream|table] with name 'foo' already exists

We can build all of this in the base classes exception if we just pass down the entity type that is being created.  So here we'd just have:

```java
public DdlCommandResult run(final MutableMetaStore metaStore) {
   registerTopic(metaStore, "stream");
  ...
}
```

And then the base class can build the same error message.   (Note, I've changed it from `createTopic` to `registerTopic` as the old name is somewhat misleading.

Reply:
Done